lib/protoparser/prometheus/parser.go

```diff
@@ -846,8 +846,10 @@ func unmarshalMetadata(dst []Metadata, s string, errLogger func(s string)) []Met
 	s = s[idx+1:]
 	idx = nextWhitespace(s)
 	if idx < 0 {
-		if errLogger != nil {
-			errLogger(fmt.Sprintf("cannot unmarshal metadata line %q: wrong exposition format", fullLine))
+		// HELP can have null content as "# HELP process_cpu_cores_available"
```

Comment:
I'm a bit confused by this comment. According to it, "HELP" with empty message is valid metadata and it should be processed by vmagent and forwarded to the remote storage.

 But in fact, it silently skipped.


 I suggest to rephrase comment. Something like:

```
 Type can't have empty value and it should be explicitly logged as error. 
 While empty HELP is valid, but there is no point to parse and store it.
```